### ISSUE TITLE
Bug - Shared notes align RTL language text to left instead of to the right

### ISSUE THREAD

**Issue by yossi-a3d** [bigbluebutton/bigbluebutton#19381]
This bug will only be shown for the role of a viewer! 
and only if Shared notes editing is locked in lock setting.

for moderators, the etherpad iframe shows the text correctly aligned to the right.

**To Reproduce**

1. enter as a moderator.
2. change to RTL language. 
3. make sure Shared notes edit is disabled in lock setting.
4. type text in Shared notes.
5. enter as a viewer. Note that the  RTL  language text is wrongly aligned to the left.
6. go back to moderator tab and enable Shared notes edit in lock setting.
7.  go back to viewer tab. Note that the text moved and is now correctly aligned to the right.

**BBB version:**
all  versions

**Comment by antobinary** [comment]
BBB 4.0 defaults to BlockNote shared notes. Any chance you could check if this works correctly, please? I just tried it and it seems to but I am not really familiar with RTL languages and would not spot if it's 100% accurate.

**Comment by yossi-a3d** [comment]
Yes, I checked, and BlockNote shared notes works well with RTL languages. The cursor positioning is accurate, and I didn't encounter any issues as long as the application interface language is set to one of the supported RTL languages.

On the other hand, writing in RTL on the whiteboard (tldraw) seems to use "push mode," which causes issues. For example, when using the text tool, a question mark at the end of a sentence jumps to the beginning (instead of staying at the end as it should in RTL).

Interestingly, if you use [tldraw](https://www.tldraw.com/) outside of BBB, RTL writing works completely fine—the cursor behaves correctly, and there are no formatting bugs. I recommend testing it directly on tldraw.com to see the difference.

(I know this is slightly off-topic for this issue, but thought it was worth mentioning).

**Comment by ramonlsouza** [comment]
Closing this issue because we confirmed that the original problem does not occur in BlockNote shared notes.

A pull request has been opened in the tldraw fork repository to address the RTL issues with the text tool: https://github.com/bigbluebutton/tldraw/pull/56
